projects/addon-doc/interfaces/page.ts

```diff
@@ -16,7 +16,7 @@ export interface TuiDocPageGroup extends TuiDocPageBase {
     readonly subPages: readonly TuiDocPage[];
 }
 
-export type RawLoaderContent = Promise<{default: string}> | string;
+export type RawLoaderContent = Promise<{default: string}> | Promise<unknown> | string;
```

Comment:
To solve this "bug" we can just change a little these lines:
https://github.com/taiga-family/taiga-ui/blob/728f07ec7e3fed0a846b540b36ac7286083ab90d/projects/demo/src/modules/components/data-list-wrapper/data-list-wrapper.component.ts#L14-L16

```ts
readonly example1: TuiDocExample = {
        TypeScript: import('./examples/1/index.ts?raw'),
        HTML: import('./examples/1/index.html?raw'),
    };
```

I am not sure that we need to expand this type with so non-strict typing.

Reply:
@vladimirpotekhin what do you think about it ?

Reply:
It would be nice to not require typing such objects. Why does this happen, is it just a webstorm getting the types wrong?

Reply:
`just a webstorm`